pages/nft-book-store/new.vue

```diff
@@ -479,9 +481,12 @@ async function submitEditedClass () {
     const p = mapPrices(prices.value)
     const price = p[0]
 
-    if (!price || !price.price) {
+    if (!price || price.price === undefined) {
       throw new Error('Please input price of edition')
     }
+    if (price.price !== 0 && price.price < MINIMAL_PRICE) {
+      throw new Error(`Price of each edition must be at least $${MINIMAL_PRICE}`)
```

Comment:
Should we just create a function to check the price since this kind of code keeps repeating?
```suggestion
      throw new Error(`Price of each edition must be at least $${MINIMAL_PRICE} or free`)
```

Reply:
the whole repo need refactoring to components instead of pages...